oshmem/mca/spml/spml.h

```diff
@@ -253,6 +253,639 @@ typedef int (*mca_spml_base_module_put_nb_fn_t)(shmem_ctx_t ctx,
                                                 int dst,
                                                 void **handle);
 
+
+
+/**
+ * The put-with-signal routines provide a method for copying data from a
+ * contiguous local data object to a data object on a specified PE and
+ * subsequently updating a remote flag to signal completion.
+ *
+ * @param ctx      A context handle specifying the context on which to perform the
+ *                 operation. When this argument is not provided, the operation is
+ *                 performed on the default context.
```

Comment:
```When this argument is not provided``` - pass NULL pointer as context?

Reply:
Referring to the SHMEM_CTX_DEFAULT constant (it is set to NULL in the code).